bigtop-packages/src/deb/phoenix/control

```diff
@@ -23,7 +23,7 @@ Homepage: http://phoenix.apache.org
 
 Package: phoenix
 Architecture: all
-Depends: zookeeper, hadoop, hadoop-mapreduce, hadoop-yarn, hbase, bigtop-utils (>= 0.7)
```

Comment:
I guess the original author added the dependency for tools like [Phoenix Map Reduce](https://phoenix.apache.org/phoenix_mr.html) using libraries of Hadoop.

While tools installed under /usr/lib/phoenix/bin uses classpath assuming hadoop, it does not assume hbase classpath because fat cliant jar contains HBase classes.

```
-cp "/etc/hbase/conf:/usr/lib/phoenix/bin/../phoenix-client-hbase-2.4-5.1.2.jar:/etc/hadoop/conf:/etc/hadoop/conf:/usr/lib/hadoop/lib/*:/usr/lib/hadoop/.//*:/usr/lib/hadoop-hdfs/./:/usr/lib/hadoop-hdfs/lib/*:/usr/lib/hadoop-hdfs/.//*:/usr/lib/hadoop-mapreduce/.//*:/usr/lib/hadoop-yarn/./:/usr/lib/hadoop-yarn/lib/*:/usr/lib/hadoop-yarn/.//*"
```
```
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.hbase.client | head -n 3
     0 Wed Jan 22 15:10:16 UTC 2020 org/apache/hadoop/hbase/client/
   570 Wed Jan 22 15:10:16 UTC 2020 org/apache/hadoop/hbase/client/RegionInfoUtil.class
  1312 Wed Jan 22 15:10:16 UTC 2020 org/apache/hadoop/hbase/client/AbstractClientScanner.class

# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.hbase | wc -l
8380
```

Reply:
If the fat jar contains Hadoop classes for running MR tools, we can remove this package dependencies. We do not need to add hbase dependency anyway.

Reply:
thanks for your advice. the phoenix-client-hbase*.jar is a thick client for phoenix, It has contained all hadoop and hbase dependencies. 
```shell
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.yarn | head -n 3
     0 Fri May 06 23:35:42 UTC 2022 org/apache/hadoop/yarn/
     0 Fri May 06 23:35:42 UTC 2022 org/apache/hadoop/yarn/api/
     0 Fri May 06 23:35:42 UTC 2022 org/apache/hadoop/yarn/api/records/
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.yarn | wc -l
2714
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.hdfs | head -n 3
     0 Fri May 06 23:29:32 UTC 2022 org/apache/hadoop/hdfs/
     0 Fri May 06 23:29:32 UTC 2022 org/apache/hadoop/hdfs/client/
   607 Fri May 06 23:29:32 UTC 2022 org/apache/hadoop/hdfs/client/package-info.class
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.hdfs | wc -l
4475
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.mapred | head -n 3
     0 Fri May 06 23:41:52 UTC 2022 org/apache/hadoop/mapreduce/
     0 Fri May 06 23:41:52 UTC 2022 org/apache/hadoop/mapreduce/lib/
     0 Fri May 06 23:41:52 UTC 2022 org/apache/hadoop/mapreduce/lib/input/
# jar tvf  /usr/lib/phoenix/phoenix-client-hbase-2.4-5.1.2.jar | grep org.apache.hadoop.mapred | wc -l
954
```
and the phoenix_util.py, I think it is for thin client like query server, however our current distros don't contain query server components. so I will remote the hbase dependency, too.